schemas/telemetry/configuration-schema.json

```diff
@@ -178,6 +179,48 @@
                 "track_native_crashes": {
                   "type": "boolean",
                   "description": "Whether native crashes are tracked"
+                },
+                "track_actions": {
+                  "type": "boolean",
+                  "description": "Whether action tracking is performed automatically"
+                },
+                "track_errors": {
+                  "type": "boolean",
+                  "description": "Whether error monitoring & crash reporting is enabled for the source platform"
+                },
+                "track_native_views": {
+                  "type": "boolean",
+                  "description": "Whether native views are tracked (for cross platform SDKs)"
+                },
+                "track_native_errors": {
```

Comment:
We already have a `track_native_crashes` key (`"Whether native crashes are tracked"`), I think this one is a duplicate?